docs/dev/Lua API.rst

```diff
@@ -1783,6 +1783,54 @@ Items module
 
   Checks whether the item is assigned to a squad.
 
+.. _lua-world:
+
+World module
+------------
+
+* ``dfhack.world.ReadPauseState()``
+
+  Returns *true* if the game is paused.
+
+* ``dfhack.world.SetPauseState(paused)``
+
+  Sets the pause state of the game.
+
+* ``dfhack.world.ReadCurrentYear()``
+
+  Returns the current game year.
+
+* ``dfhack.world.ReadCurrentTick()``
+
+  Returns the current game tick.
```

Comment:
should note that this is the number of ticks since the start of the current game year. it starts over every year. The "current game tick" is `df.global.world.frame_counter`